templates/common/infra/bicep/core/database/mysql/mysql-database.bicep

```diff
@@ -0,0 +1,18 @@
+param name string = ''
+param mysqlServerName string = ''
```

Comment:
I think mysql should be mySql

Reply:
it should be `mySQL` based on its official name (`MySQL`), so I changed it to `mySQL`.